packages/rke2-canal/charts/values.yaml

```diff
@@ -25,15 +25,15 @@ calico:
   # CNI installation image.
   cniImage:
     repository: rancher/hardened-calico
-    tag: v3.20.1-build20211006
+    tag: v3.20.1-build20211030
```

Comment:
@raulcabello not https://github.com/rancher/image-build-calico/releases/tag/v3.20.2-build20211115 ?